R/install.R

```diff
@@ -135,7 +135,12 @@ install <- function(packages = NULL,
   }
 
   # get and resolve the packages / remotes to be installed
-  remotes <- packages %||% renv_project_remotes(project, dependencies)
+  if (length(packages) == 0) {
+    remotes <- renv_project_remotes(project, dependencies)
+  } else {
+    remotes <- packages
+  }
```

Comment:
```suggestion
  remotes <- if (length(packages) == 0) {
    renv_project_remotes(project, dependencies)
  } else {
    packages
  }
```

Not sure how you feel, but I usually prefer assigning the result of the `if` expression if all branches are just assigning the same variable.

Reply:
We could also change the code here: https://github.com/rstudio/renv/pull/1304/files?w=1#diff-76ebee0a54b80584fcf1eff0cdf55ef6f17e47a6d80cb807b5939e8e50cac32cR110

Since that would help us preserve the NULL-ness of the incoming argument.

Reply:
Oh yeah, let's change it there.